docs/release_docs/REAN_Platform_Deployment_Workflows.md

```diff
@@ -0,0 +1,97 @@
+# REAN Platform Deployment workflows
+
+## PR-CI-CD
+mode of trigger: ```Automated```
+
+ PR Workflow is triggered automatically whenever a PR with source branch as a feature/* branch is created against the target branch as develop branch.
+ ![PR-ci-cd_workflow](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/pr-ci-cd_workflow.png?raw=true)
+ ![pr](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/Pr-ci-cd_example.png?raw=true)
+ 
```

Comment:
I added a comment add some headers to distinguish both images, which is marked as resolved, but I still don't see the changes here. Currently, it looks like its a single image
![image](https://github.com/REAN-Foundation/reancare-service/assets/30074835/e728d246-07bc-42fc-8f1c-6e838fc6b3de)